cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2369,6 +2369,593 @@ endmacro()
+                                                          ${_ABSL_STATIC_LIBRARY})
+      list(APPEND ABSL_BUILD_BYPRODUCTS ${_ABSL_STATIC_LIBRARY})
+    endforeach()
+    foreach(_ABSL_LIB ${_ABSL_INTERFACE_LIBS})
+      add_library(absl::${_ABSL_LIB} INTERFACE IMPORTED)
+      set_target_properties(absl::${_ABSL_LIB} PROPERTIES INTERFACE_INCLUDE_DIRECTORIES
+                                                          "${ABSL_PREFIX}/include")
+    endforeach()
+
+    # Extracted the dependency information using the Abseil pkg-config files:
+    #   grep Requires $PREFIX/pkgconfig/absl_*.pc | \
+    #   sed -e 's;.*/absl_;set_target_properties(absl::;' \
+    #       -e 's/.pc:Requires:/ PROPERTIES INTERFACE_LINK_LIBRARIES "/' \
+    #       -e 's/ = 20210324,//g' \
+    #       -e 's/ = 20210324//g' \
+    #       -e 's/ absl_/;absl::/g' \
+    #       -e 's/$/")/' -e 's/";/"/' | \
+    #   grep -v '"")'
+    set_target_properties(absl::algorithm_container
+                          PROPERTIES INTERFACE_LINK_LIBRARIES
+                                     "absl::algorithm;absl::core_headers;absl::meta")
+    set_target_properties(absl::algorithm PROPERTIES INTERFACE_LINK_LIBRARIES
+                                                     "absl::config")
+    set_target_properties(absl::any
+                          PROPERTIES INTERFACE_LINK_LIBRARIES
+                                     "absl::bad_any_cast;absl::config;absl::core_headers;absl::fast_type_id;absl::type_traits;absl::utility"
+    )
+    set_target_properties(absl::atomic_hook PROPERTIES INTERFACE_LINK_LIBRARIES
+                                                       "absl::config;absl::core_headers")
+    set_target_properties(absl::bad_any_cast_impl
+                          PROPERTIES INTERFACE_LINK_LIBRARIES
+                                     "absl::config;absl::raw_logging_internal")
+    set_target_properties(absl::bad_any_cast
+                          PROPERTIES INTERFACE_LINK_LIBRARIES
+                                     "absl::bad_any_cast_impl;absl::config")
+    set_target_properties(absl::bad_optional_access
+                          PROPERTIES INTERFACE_LINK_LIBRARIES
+                                     "absl::config;absl::raw_logging_internal")
+    set_target_properties(absl::bad_variant_access
+                          PROPERTIES INTERFACE_LINK_LIBRARIES
+                                     "absl::config;absl::raw_logging_internal")
+    set_target_properties(absl::base_internal PROPERTIES INTERFACE_LINK_LIBRARIES
+                                                         "absl::config;absl::type_traits")
+    set_target_properties(absl::base
+                          PROPERTIES INTERFACE_LINK_LIBRARIES
+                                     "absl::atomic_hook;absl::base_internal;absl::config;absl::core_headers;absl::dynamic_annotations;absl::log_severity;absl::raw_logging_internal;absl::spinlock_wait;absl::type_traits"
```

Comment:
This list is very long.
How about using `set_property(TARGET absl:... PROPERTY INTERFACE_LINK_LIBRARIES absl::... absl:: ...)` https://cmake.org/cmake/help/latest/command/set_property.html#command:set_property ?

Reply:
Thanks, that is a good idea.  Let me test that locally while the other builds run.

Reply:
Done.